packages/slate/src/interfaces/operation.ts

```diff
@@ -1,23 +1,35 @@
 import { Node, Path, Range } from '..'
 import isPlainObject from 'is-plain-object'
 
+export enum OperationType {
+  InsertNode = 'insert_node',
+  InsertText = 'insert_text',
+  MergeNode = 'merge_node',
+  MoveNode = 'move_node',
+  RemoveNode = 'remove_node',
+  RemoveText = 'remove_text',
+  SetNode = 'set_node',
+  SetSelection = 'set_selection',
+  SplitNode = 'split_node',
+}
```

Comment:
If is possible to use a [const enum](https://www.typescriptlang.org/docs/handbook/enums.html) here so there's no extra generated code, as was the case with the string literal types that were previously used?